notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        anim = xvid(fc, 15, figsize=4, mask=land_mask)
</pre><p>Is there a way to label the colorbar with something like "Sea-ice concentration fraction"? Maybe a feature request to the IceNet library?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>Added option to do this, now the plots include the colourbar label.</p>